python-evaluator-transform/src/main/java/co/cask/hydrator/transforms/PythonEvaluator.java

```diff
@@ -174,7 +171,7 @@ public void transform(StructuredRecord input, final Emitter<StructuredRecord> em
       Py.runCode(compiledScript, interpreter.getLocals(), interpreter.getLocals());
 
     } catch (Exception e) {
-      throw new IllegalArgumentException("Could not transform input: " + e.getMessage(), e);
+      throw new IllegalArgumentException("Could not transform input.\n" + StringUtils.stringifyException(e));
```

Comment:
Lets add a comment here to explain that PyException only exposes the actual cause if .printStackTrace() is called on it, which is why its not logged by ETLMapper